### ISSUE TITLE
Autorest 2.x node 14 support 

### ISSUE THREAD

**Issue by tjprescott** [Azure/autorest#3983]
**Describe the bug**
```
Please provide as much information as you can. This would be: 
Generating: specification/cognitiveservices/data-plane/Face/readme.md
Command: autorest readme.me --python --python-mode=update --python-sdks-folder=<path> --no-async --multiapi --use=@microsoft.azure/autorest.python@~4.0.71 --version=V2 --keepVersionFile
Using Node version 14.16.0
```
This yields an error:
![image](https://user-images.githubusercontent.com/5723682/110979606-ab856800-8319-11eb-9e4d-2b5fa02fdba5.png)

**Expected behavior**
This should generate.

After much frustration, it I changed my Node version to 13.12.0 and it worked fine.

**Comment by timotheeguerin** [comment]
Believe this is not an issue with the latest version of core. I think the problem is that by default when using track1 it loads autorest core `2.x`. @iscai-msft I am not very familiar with the track1 story, is track1 is meant to be used with autorest `3.x` ?

**Comment by iscai-msft** [comment]
talked to @tjprescott offline, issue turned out to be his node version. Downgrading from node 14 to node 13.12.0 fixed the issue. Closing

**Comment by iscai-msft** [comment]
oops should've read @tjprescott issue better, he is actually asking for support for node 14 @timotheeguerin

**Comment by timotheeguerin** [comment]
Yeah, as this only applies with autorest v2 and track1 python generator I don't think this is worth fixing.

**Comment by tjprescott** [comment]
Well I specifically went to 14.16 because of an autorest warning, so if there are scenarios where it doesn't work, then we should definitely call that out. cc @lmazuel

**Comment by timotheeguerin** [comment]
Do you remember the warning you got? and which version of node you were previously using?

**Comment by tjprescott** [comment]
I recall the warning before was something like "Autorest has only been tested up to Node version BLAH". I don't see it these days, so maybe it went away (or I am using a version that it's happy with).

**Comment by timotheeguerin** [comment]
I see, yeah autorest cli has this warning `WARNING: AutoRest has not been tested with Node versions greater than v14.`
Might have been using node 15 before

**Comment by tjprescott** [comment]
@timotheeguerin yes, I think that's right. That's why I went down to 14, which did not work for me when generated the track 1 SDK.

**Comment by timotheeguerin** [comment]
Was it working with node 15 or you didn't try track1?

**Comment by tjprescott** [comment]
I had other problems with Autorest back (as you may recall :grin:) so I downgraded per the warning thinking that might fix the issue. I haven't tried the track1 stuff with Node 15.

**Comment by timotheeguerin** [comment]
Published a new v2 version `2.0.4419`
